src/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProvider.php

```diff
@@ -384,7 +384,7 @@ private function extractStockData(ProductForEditing $productForEditing, ShopCons
             'options' => [
                 'stock_location' => $stockInformation->getLocation(),
                 'low_stock_threshold' => $stockInformation->getLowStockThreshold(),
-                'low_stock_alert' => $stockInformation->isLowStockAlertEnabled(),
+                'disabling_switch_low_stock_threshold' => $stockInformation->isLowStockAlertEnabled(),
```

Comment:
Ho nice catch! I'll probably have to fix this for combination form as well, there's probably the same issue